lib/pymedphys/_dicom/connect/listen.py

```diff
@@ -28,11 +28,11 @@ class DicomListener(DicomConnectBase):
     """
 
     def __init__(self, storage_directory=None, on_released_callback=None, **kwargs):
-        """Create and instance of a Dicom Listener
+        """Create and instance of a DICOM Listener
 
         Parameters
         ----------
-        storage_directory : [type], optional
+        storage_directory : pathlib.Path/str, optional
```

Comment:
```suggestion
        storage_directory : pathlib.Path or str, optional
```

See https://numpydoc.readthedocs.io/en/latest/format.html